.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: Continuous Integration 
+
+on: [ pull_request ]
```

Comment:
This should run on a fixed schedule (daily?) so that we can detect any breaking changes in Cirq. The idea is to ensure that if Cirq updates which breaks cirq-on-iqm, we should be able to detect that early.

Reply:
Added schedule to run tests every midnight (UTC).